migration/sql-files/001-audit-logs.sql

```diff
@@ -0,0 +1,25 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- event types
+CREATE TABLE event_type (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
```

Comment:
I would rename to `event_type_id`, it reduces ambiguity in joined queries

Reply:
ok, I see, so we would join table with the following syntax: 
````
select ... join on event_type.event_type_id = audit_log.event_type_id ... 
```` 
rather than 
````
select ... join on event_type.id = audit_log.event_type_id ... 
````
I initially found that `event_type.event_type_id` was redundant, but I can see how `event_type.event_type_id = audit_log.event_type_id` avoids any ambiguity, indeed.